extensions/amp-story-interactive/amp-story-interactive.md

```diff
@@ -1,10 +1,310 @@
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+[tip type="important"]Change image sources to ampproject/amphtml/master!!![/tip]
+
+The amp-story-interactive component provides a set of experiences, such as quizzes or polls, for users in [Web stories](https://amp.dev/documentation/guides-and-tutorials/start/create_successful_stories/?format=stories). Interactive experiences provided by amp-story-interactive can integrate into tools or editors and support analytics.
+
+<div layout="container" width="3" height="2">
+<div style="width:32%;display:inline-block"><amp-img src="https://github.com/mszylkowski/amphtml/raw/interactive_docs/extensions/amp-story-interactive/img/quiz-art.png" layout="responsive" width="200" height="350"/></div>
+<div style="width:32%;display:inline-block"><amp-img src="https://github.com/mszylkowski/amphtml/raw/interactive_docs/extensions/amp-story-interactive/img/animal-poll.png" layout="responsive" width="200" height="350"/></div>
+<div style="width:32%;display:inline-block"><amp-img src="https://github.com/mszylkowski/amphtml/raw/interactive_docs/extensions/amp-story-interactive/img/results-art.png" layout="responsive" width="200" height="350"/></div>
+</div>
+
+## Interactive experiences
+
+The amp-story-interactive extension encompasses a set of interactive experiences. Specify an interactive experience by defining one of the elements below. For best results, only use one element per amp-story-page.
+
+If you want to see all the components in action, check out the [example story](/documentation/examples/components/amp-story-interactive-poll/story#page=title-components)
+
+### amp-story-interactive-binary-poll
+
+The amp-story-interactive-binary-poll element provides a two option voting user interface. Users may select one of two valid options. When selected, the highlighted option fills the container and displays the total percentage of votes.
+
+Is well suited to be used without amp-story-interactive-results, and can optionally have a prompt.
+
+<amp-img src="https://github.com/mszylkowski/amphtml/raw/interactive_docs/extensions/amp-story-interactive/img/binary-poll-raw.png" layout="intrinsic" width="400" height="230">
+
+```html
+<amp-story-interactive-binary-poll
+  id="pizza-binary-poll"
+  endpoint="https://endpoint.com/v1/interactives"
+  prompt-text="Like Pizza?"
+  option-1-text="Yes"
+  option-1-confetti="🍕"
+  option-2-text="No"
+  option-2-confetti="🤢"
+>
+</amp-story-interactive-binary-poll>
+```
+
+### amp-story-interactive-poll
+
+The amp-story-interactive-poll element provides a voting experience with 2-4 options displayed vertically, where all options are valid. When selected, each option displays the total percentage of votes.
+
+Can be paired up with amp-story-interactive-results to display different categories based on the answers to polls, but can be used independently as well. It is encouraged to add a prompt for extra context.
```

Comment:
Changed for "Add a poll for extra content" so we don't say we encourage it, even though most people will likely use it. The way to add a prompt is to use `prompt-text` as it can be seen in the attributes section. The examples also have the prompt-text attribute